Vision/src/Annotation/SafeSearch.php

```diff
@@ -76,6 +76,16 @@
  *
  *     @return string
  * }
+ * @method racy() {
+ *     Racy likelihood.
+ *
+ *     Example:
+ *     ```
+ *     echo $safeSearch->racy();
```

Comment:
This example needs to be tested as well. Refer to `Google\Cloud\Vision\Tests\Snippet\Annotation\SafeSearchTest::testAdult()` for a guide to testing snippet in magic methods.

Reply:
https://github.com/RobLoach/google-cloud-php/blob/patch-1/Vision/tests/Snippet/Annotation/SafeSearchTest.php#L113-L120